modules/admin_manual/nav.adoc

```diff
@@ -155,6 +173,9 @@
 *** User Management
 **** xref:enterprise/user_management/user_auth_shibboleth.adoc[Shibboleth Integration]
 **** xref:enterprise/user_management/saml_2.0_sso.adoc[SAML 2.0 Based SSO]
-** xref:document_classification/index.adoc[Document Classification]
```

Comment:
Why this change?

Reply:
very easy,
beause this was the only item not having a drop down
also trouble shooting with a single entry has a drop down
harmonizing layout

Reply:
I don't quite understand the second point. Can you further clarify?

Reply:
```
** Document Classification
*** xref:document_classification/index.adoc[Document Classification and Policy Enforcement]

** Troubleshooting
*** xref:troubleshooting/providing_logs_and_config_files.adoc[Retrieve Log Files and Configuration Settings]
```

Reply:
![image](https://user-images.githubusercontent.com/3321281/57318071-c70a0100-70f9-11e9-970f-e3aa3616c318.png)